packit_service/worker/handlers/fedmsg_handlers.py

```diff
@@ -238,6 +238,7 @@ def run(self):
             logger.info(msg)
             return HandlerResults(success=True, details={"msg": msg})
 
+        build.set_end_time(datetime.utcnow())
```

Comment:
With `utcnow`, if the celery task waits in the queue for a worker, the timestamp will be quite far from reality.

I'm okay to merge this as it is now, but should get the timestamp field in the message and use that one in the future.

Do you need help with that?

Reply:
Do you mean to change it firstly in the `packit-service-fedmsg` repo as you described above and use that value?

Reply:
Yes, that's what I meant.